src/test/java/com/stormeye/evaluation/StateGetAccountInfoStepDefinitions.java

```diff
@@ -0,0 +1,100 @@
+package com.stormeye.evaluation;
+
+import com.casper.sdk.identifier.block.BlockIdentifier;
+import com.casper.sdk.identifier.block.HashBlockIdentifier;
+import com.casper.sdk.model.account.AccountData;
+import com.casper.sdk.model.block.JsonBlockData;
+import com.casper.sdk.model.key.PublicKey;
+import com.stormeye.utils.AssetUtils;
+import com.stormeye.utils.CasperClientProvider;
+import com.stormeye.utils.ParameterMap;
+import com.syntifi.crypto.key.AbstractPublicKey;
+import com.syntifi.crypto.key.Ed25519PrivateKey;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URL;
+
+import static com.stormeye.evaluation.StepConstants.STATE_ACCOUNT_INFO;
+import static com.stormeye.utils.NctlUtils.getAccountHash;
+import static com.stormeye.utils.NctlUtils.getAccountMainPurse;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.notNullValue;
+
+/**
+ * Step definitions for the state_get_account_info RPC method.
+ *
+ * @author ian@meywood.com
+ */
+public class StateGetAccountInfoStepDefinitions {
+
+    private static final ParameterMap parameterMap = ParameterMap.getInstance();
+    private static final Logger logger = LoggerFactory.getLogger(StateGetAccountInfoStepDefinitions.class);
+
+    @Given("that the state_get_account_info RCP method is invoked")
+    public void thatTheStateGetAccountInfoRcpMethodIsInvoked() throws IOException {
+
+        logger.info("Given that the state_get_account_info RCP method is invoked");
+
+        final String hexPublicKey = getUserOneHexPublicKey();
+        final JsonBlockData block = CasperClientProvider.getInstance().getCasperService().getBlock();
+        final BlockIdentifier identifier = new HashBlockIdentifier(block.getBlock().getHash().toString());
+
+        final AccountData stateAccountInfo = CasperClientProvider.getInstance().getCasperService().getStateAccountInfo(hexPublicKey, identifier);
+        parameterMap.put(STATE_ACCOUNT_INFO, stateAccountInfo);
```

Comment:
query NCTL here and uses as EXPECTED_RESULTS

nctl-view-chain-account account-key=hexPublicKey